contracts/abstracts/Controlable.sol

```diff
@@ -44,21 +54,21 @@ abstract contract Controlable is AccessControlUpgradeable {
     return true;
   }
 
-  function _changeTransactionFee(uint32 transactionFee_) internal returns (bool success) {
+  function _changeTransactionFee(uint32 transactionFee_) internal onlyAdmin returns (bool success) {
     uint32 oldFee = _transactionFee;
     _transactionFee = transactionFee_;
     emit TransactionFeeChanged(oldFee, transactionFee_);
     return true;
   }
 
   // Treasury
-  function _withdrawTransactionFee() internal returns (bool success) {
+  function _withdrawTransactionFee() internal onlyTreasury returns (bool success) {
     require(_token.transfer(msg.sender, _accumulatedTransactionFee), 'Controlable: Transfer failed');
     return true;
   }
 
   // Moderator
-  function _blacklistToken(address tokenContract, uint256 tokenId, bool isBlacklisted) internal returns (bool success) {
+  function _blacklistToken(address tokenContract, uint256 tokenId, bool isBlacklisted) internal onlyModerator returns (bool success) {
```

Comment:
since you already added the modifier on the external function, we don't need to add it to the internal function